CONTRIBUTING.md

```diff
@@ -183,6 +183,16 @@ We like [this style guide][guide].
 
 [guide]: https://github.com/tibbe/haskell-style-guide/blob/master/haskell-style.md
 
+GitHub conventions
+-------------------
+
+Each `Cabal`/`cabal-install` release has a corresponding GitHub Project and milestone.
+A ticket is included in a release's project if the release manager is tenatively
+planning on including a fix for the ticket in the release.
```

Comment:
(add -- i.e. if they are actively seeking someone to work on the ticket)

Reply:
only the release manager can do it, then? do we have release manager now? If we mention that role, would not be good to talk about who is or are the individuals in docs?

Reply:
I think it's the release manager *for that release*. But we are more distributed/chaotic than that, so the release manager is not known until the night of the release. So, perhaps, "release managers" meaning all maintainers?